src/tests/multihost/alltests/test_default_debug_level.py

```diff
@@ -0,0 +1,150 @@
+"""Automation for default debug level
+
+:requirement: SSSD - Default debug level
+:casecomponent: sssd
+:subsystemteam: sst_idm_sssd
+:upstream: yes
+"""
+
+from __future__ import print_function
+import re
+import time
+import pytest
+from sssd.testlib.common.utils import sssdTools
+from sssd.testlib.common.expect import pexpect_ssh
+from constants import ds_instance_name
+
+
+@pytest.mark.usefixtures('setup_sssd', 'create_posix_usersgroups')
+@pytest.mark.defaultdebuglevel
+@pytest.mark.tier1_2
+class TestDefaultDebugLevel(object):
+    """ Check sssd default debug level """
+    def test_0001_check_default_debug_level(self, multihost, backupsssdconf):
+        """
+        :title: default debug logs: Check default debug level when sssd start
+         successfully
+        :id: 1f38b560-27dc-4144-895d-e667420b0467
+        """
+        section = "domain/%s" % ds_instance_name
+        domain_params = {'debug_level': ''}
+        tools = sssdTools(multihost.client[0])
+        tools.sssd_conf(section, domain_params, action='delete')
+        # stop sssd, delete sssd logs and cache, start sssd
+        tools.clear_sssd_cache()
+        log_list = ['sssd', f'sssd_{ds_instance_name}',
+                   'sssd_nss', 'sssd_pam']
+        for log_filename in log_list:
+            log = '/var/log/sssd/%s.log' % log_filename
+            log_str = multihost.client[0].get_file_contents(log).decode(
+                'utf-8')
+            pattern1 = re.compile(r'Starting with debug level = 0x0070')
+            default_debug = pattern1.search(log_str)
+            log_split = log_str.split("\n")
+            for index in range(len(log_split)-1):
+                log_single_line = log_split[index]
+                pattern2 = re.compile(r'(0x\w+)')
+                debug_str = pattern2.search(log_single_line)
+                if debug_str.group() != '0x1f7c0':
+                    assert False
+        assert default_debug is not None
```

Comment:
`default_debug` is set within the loop (every iteration), but checked outside (once). Is this the intent?

Reply:
Fixed